Movies_mvp/Movies_mvp/MovieDetailViewController/Model/MovieDetail.swift

```diff
@@ -1,27 +1,29 @@
 // MovieDetail.swift
 // Copyright © RoadMap. All rights reserved.
 
+import SwiftyJSON
+
 /// Модель детального описания фильма
-struct MovieDetail: Decodable {
+struct MovieDetail {
     let backdropPath: String
     let posterPath: String?
     let title: String
     let runtime: Int?
     let voteAverage: Double
     let imdbId: String?
     let releaseDateString: String
-    let genres: [Genre]
+    let genres: [String]
```

Comment:
```suggestion
    /// жанры
    let genres: [String]
```
возможно скажут подписать, опциональное замечание